### PR TITLE
10255 - Bug fix for Account accessType change premium to GovN

### PR DESCRIPTION
**Pull request by karthik-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/10255

*Description of changes:*

- Add conditions to display premium accounts

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/account-settings/account-info/AccountAccessType.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
                   <span data-test="txt-selected-access-type">{{ selectedAccessType === AccessType.GOVN ? 'Government agency (other than BC provincial)' : 'Regular Access'}}</span>
                 </div>
                 <div
-                  v-if="canChangeAccessType"
+                  v-if="canChangeAccessType && selectedAccessType !== AccessType.GOVN"
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/account-settings/account-info/AccountAccessType.vue:23]: Is this condition to enable the Change button? If so, shouldn't this check for GOVM too ?

Comment (karthik-aot) [reply auth-web/src/components/auth/account-settings/account-info/AccountAccessType.vue:23]: The parent component only filters the following access types to be shown
image

Comment (karthik-aot) [reply auth-web/src/components/auth/account-settings/account-info/AccountAccessType.vue]: Refactored the code accordingly

File: auth-web/src/components/auth/account-settings/account-info/AccountAccessType.vue [hunk 2]
```diff
@@ -122,6 +122,25 @@ export default class AccountAccessType extends Vue {
     return this.currentOrgPaymentType && this.currentOrgPaymentType === PaymentTypes.PAD
   }
 
+  public get isChangeButtonEnabled (): boolean {
+    let isChangeButtonEnabled: boolean = this.organization.orgType === Account.PREMIUM // Only Premium accounts can be edited
```
Comment (shabeeb-aot) [inline auth-web/src/components/auth/account-settings/account-info/AccountAccessType.vue:126]: use different name for function and variables

**Review by shabeeb-aot — APPROVED** [review]

**Review by sumesh-aot — APPROVED** [review]

**Pull request merged**